backend/movies/views.py

```diff
@@ -20,11 +22,14 @@ def create(self, request):
     if not serializer.is_valid():
       return Response(serializer.errors, status=400)
     serializer.validated_data['admin'] = request.user
+    admin = TheaterAdmin.objects.get(user_ptr_id=request.user.id)
+    serializer.validated_data['theater'] = admin.theater
```

Comment:
Mislim da nije dobro da se menjaju validirani podaci. Dodaj u serializator PrimaryKeyRelatedField i onda pošalji u sklopu request-a id pozorižta/bioskopa. Možeš pogledati TheaterAdminSerializator, tamo je dodato to polje.